netsnmp/__init__.py

```diff
@@ -112,6 +111,7 @@ def get(self, oids):
         """
         # Define list to be populated by C API get()
         responses = []
+        self._next = False
```

Comment:
however, without this, if use getnext() once, it will change this flag and after a getnext(), all subsequent get() calls will happen as getnext() because both calls the same function that checks that flag and it won't be set back to its default state.